test/unit/controllers/declaration/TotalNumberOfItemsControllerSpec.scala

```diff
@@ -103,7 +103,7 @@ class TotalNumberOfItemsControllerSpec extends ControllerSpec with OptionValues
 
         await(result) mustBe aRedirectToTheNextPage
         thePageNavigatedTo mustBe controllers.declaration.routes.TotalPackageQuantityController.displayPage()
-        verify(mockTotalNumberOfItemsPage, times(0)).apply(any(), any())(any(), any())
+        verify(mockTotalNumberOfItemsPage, times(0)).apply(any(), any(), any())(any(), any())
```

Comment:
Nit:  I've tended to create a helper method for these page verify tests with only the expected number of interactions as a variable